cmd/tilde-config/gnss.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"fmt"
+	"math"
+
+	"github.com/GeoNet/delta/meta"
+)
+
+func (t *Tilde) Gnss(set *meta.Set) error {
+	var stns []Station
+
+	// all marks are GNSS stations
+	for _, m := range set.Marks() {
+		stns = append(stns, Station{
+			Code:        m.Code,
+			Description: m.Name,
+			Start:       toTimePtr(m.Start),
+			End:         toTimePtr(m.End),
+			Latitude:    toFloat(fmt.Sprintf("%0.4f", m.Latitude)),
+			Longitude:   toFloat(fmt.Sprintf("%0.4f", m.Longitude)),
+			Elevation:   toFloat(fmt.Sprintf("%0.0f", math.Round(m.Elevation))),
+		})
+	}
+
+	// update domains
+	t.Domains = append(t.Domains, Domain{
+		Name:        "gnss",
+		Description: "Global Navigation Satellite System Sensors",
```

Comment:
Hey @wilsonjord ! After a discussion with @elidana  and @rumachan I still would like to propose changes to the domain description and go with `"Global Navigation Satellite System Stations"`. Sorry for the slow review!

Reply:
"station" is the only terminology that is not used in tilde domain descriptions (see https://tilde.geonet.org.nz/v4/dataSummary/) and in the domain model it refers to the generalised "location" where time series data are available (can be from multiple `sensors` for some domains). So maybe let's just stick to `Global Navigation Satellite System` as a compromise?

Reply:
@elidana @aleks-spes could I get a final confirmation on the last question from @elidana ?

Reply:
i personally vote for `Global Navigation Satellite System` as a compromise